x-pack/plugins/observability/public/components/app/burn_rate_rule_editor/burn_rate_rule_editor.tsx

```diff
@@ -13,7 +13,7 @@ import { i18n } from '@kbn/i18n';
 import { SLOResponse } from '@kbn/slo-schema';
 
 import { toDuration, toMinutes } from '../../../utils/slo/duration';
-import { useFetchSloDetails } from '../../../hooks/slo/use_fetch_slo_details';
+import { useFetchSloDetails } from '../../../hooks/slo/use_fetch_slo_details_rq';
```

Comment:
Are we planning to rename the file without the `_rq` suffix and remove the previous hooks?

Reply:
Yes :) Just updated the PR